index.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-unused-vars */
+import { setupNavigation } from './modules/navbar.js';
+import BookShelf from './modules/bookshelf.js';
+import { DateTime } from './node_modules/luxon/src/luxon.js';
```

Comment:
- [ ] [OPTIONAL] I highly recommend adding the `luxon.js` file to your components folder instead, this way you don't need to install luxon every time you clone the project

Reply:
Hello @Ismailco,
Please how do I add the `luxon.js` file to my components folder?
I tried something earlier, I created a luxon.min.js file and copied the  min version of the code into it, but later changed to what I have in the project now. Please give me more guidelines on how to do that.

Thanks

Reply:
Hello @Ismailco,
Please how do I add the `luxon.js` file to my components folder?
I tried something earlier, I created a luxon.min.js file and copied the  min version of the code into it, but later changed to what I have in the project now. Please give me more guidelines on how to do that.

Thanks

Reply:
Hi @christianonoh 👋🏼 what the previous reviewer meant about configuring  a local version of luxon in  your local project you can achieve that by following the steps below:

- Create a local luxon javascript file inside your project directory.
- Copy the minified version from the documentation link under [ES6](https://moment.github.io/luxon/#/install).
- Then import it by `import { DateTime } from "luxon";` 👍🏼 

You can find different methods to use in [Quick tour](https://moment.github.io/luxon/#/tour) 🚀